infrastructure/terraform/keep-prd/variables.tf

```diff
@@ -0,0 +1,87 @@
+# env vars
+variable "gcp_thesis_org_id" {
+  description = "The ID for the organization the project will be created under. Local ENV VAR."
```

Comment:
When we say “local” here we literally mean “on the machine running `terraform apply`”?

Reply:
Aye.